src/main/scala/scalaz/netty/Server.scala

```diff
@@ -126,10 +126,9 @@ private[netty] object Server {
 
     //val server = new Server(bossGroup, config.limit)
     val bootstrap = new ServerBootstrap
-    
+
     val serverQueue = async.boundedQueue[(InetSocketAddress, Process[Task, Exchange[ByteVector, ByteVector]])](config.limit)
-    val handlerQueue = async.boundedQueue[ByteVector](config.limit)
```

Comment:
Actually, slightly more subtle than that (though I do think that connection closure probably would screw things up).  With every connection sharing the same queue, incoming connections get each others data in non-deterministic order!